exchange/websocket/manager.go

```diff
@@ -739,51 +739,51 @@ func (m *Manager) CanUseAuthenticatedWebsocketForWrapper() bool {
 }
 
 // SetWebsocketURL sets websocket URL and can refresh underlying connections
-func (m *Manager) SetWebsocketURL(url string, auth, reconnect bool) error {
+func (m *Manager) SetWebsocketURL(wsURL string, auth, reconnect bool) error {
```

Comment:
🐒  No other url in play here, so:
```suggestion
func (m *Manager) SetWebsocketURL(u string, auth, reconnect bool) error {
```